test/issue1396-pre-post-run-commands/dub.sdl

```diff
@@ -0,0 +1,3 @@
+name "test"
+preRunCommands "echo \"pre-run\n\" >> test.txt"
```

Comment:
Not sure why you need the newline

Reply:
My impression was, the newline is needed for the grep command. But that is not true. I removed them.